### PR TITLE
Adding additional metric for UDP Protocol

### PR DESCRIPTION
**Pull request by hanamantagoudvk**
a) Loss of packet in percentage added
b) Percentile calculation corrected

### CODE REVIEW

**Comment by hanamantagoudvk** [comment]
@wojtek-t @mm4tt : could you please review this PR ?

File: clusterloader2/pkg/measurement/common/network/network_performance_measurement.go [hunk 1]
```diff
@@ -358,6 +358,11 @@ func (npm *networkPerformanceMeasurement) calculateMetricDataValue(dataElem *mea
 	case manyToMany:
 		for _, metricResponse = range npm.metricVal {
 			if len(metricResponse) > 0 {
+				// Sometimes iperf gives negative values for latency. As short-term fix
+				// we are considering them as zero.
+				if metricIndex == udpLatencyAverage && metricResponse[metricIndex] < 0 {
+					continue
```
Comment (marseel) [inline clusterloader2/pkg/measurement/common/network/network_performance_measurement.go:364]: I see that later on you are computing percentile based on `aggregatePodPairMetrics ` array. Here you are skipping adding results for udpLatencyAverage if latency is lower than zero. If you want to assume that negative latency is zero as short term fix, don't you want to append this zero to array for percentile computation? 
Could you clarify maybe what negative latency means (I can imagine that adding zero/not adding zero to array might depend on it).

Comment (hanamantagoudvk) [reply clusterloader2/pkg/measurement/common/network/network_performance_measurement.go:364]: Hi @marseel , you can fine more context to this PR in the issue we raised https://github.com/kubernetes/perf-tests/issues/1818

**Pull request merged**